db/migrate/20181107172409_remove_postal_address_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePostalAddressFromUsers < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :users, :postal_address, :text
```

Comment:
Should we care about existing users having set up a `postal_address` already?  Would there be a way to migrate their data to the new schema?